locale/en/api.po

```diff
@@ -329,5 +329,8 @@ msgstr "The submission for this review assignment could not be found."
 msgid "api.reviews.assignments.invalidReviewer"
 msgstr "The reviewer for the assignment could not be found"
 
+msgid "api.review.assignments.invalidInvitationResponse"
+msgstr "The review invitation response is invalid"
```

Comment:
Since there are only two valid cases, we can probably include them as part of the error (i.e. only `accept` and `decline` are accepted values).